File: Modul 2/SOAL 10/PRAK110-2210817120003-IKHTIARA FAKHRUNISA.py
import math
alas = 5
tinggi = 12
miring = round(math.sqrt(alas*alas+tinggi*tinggi))
k = alas + tinggi + miring
l = 1/2 * alas * tinggi

print("Diketahui :")
print("Alas = ", alas,"cm")
print("Tinggi = ", tinggi,"cm\n")
print("Jawab :")
print("Sisi A = ",tinggi,"cm")
print("Sisi B = ",round(miring),"cm")
print("Sisi C = ",alas,"cm")
print("Keliling = ",round(k),"cm")
print("Luas = ",round(l),"cm")

File: Modul 2/SOAL 10/PRAK110-2210817120003-IKHTIARAFAKHRUNISA.c
#include <stdio.h>
#include <math.h>

int main (){
    float alas,tinggi,miring,k,l;
    alas = 5;
    tinggi = 12;
    miring = sqrt((alas*alas)+(tinggi*tinggi));
    k = alas + tinggi + miring;
    l = 0.5 * alas * tinggi;

    printf("Diketahui : \n");
    printf("Alas = %.f cm\n",alas);
    printf("Tinggi = %.f cm\n\n", tinggi);
    printf("Jawab : \n");
    printf("Sisi A = %.f cm\n",tinggi);
    printf("Sisi B = %.f cm\n",miring);
    printf("Sisi C = %.f cm\n",alas);
    printf("Keliling = %.f cm\n",k);
    printf("Luas = %.f cm\n",l);
    return 0;
}
